frontend/src/metabase/home/components/EmbedHomepage/tests/common.unit.spec.tsx

```diff
@@ -167,7 +167,7 @@ describe("EmbedHomepage (OSS)", () => {
       const lastCall = getLastHomepageSettingSettingCall();
 
       const body = await lastCall?.request?.json();
-      expect(body).toEqual({ value: "dismiss-run-into-issues" });
+      expect(body).toEqual({ value: "dismissed-run-into-issues" });
```

Comment:
Not sure how this happened, but the correct was was always dismiss*ed* https://github.com/metabase/metabase/blob/fa58c1345f7b96f0c29ef7aedc71ce3297e362bd/frontend/src/metabase-types/api/settings.ts#L236